content/it/cloud-computing.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Cloud Computing
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+Il cloud computing è un modello che offre risorse di calcolo come CPU, rete e capacità del disco su richiesta su Internet. Il cloud computing offre agli utenti la possibilità di accedere e utilizzare la potenza di calcolo in una posizione fisica remota. I provider cloud come AWS, GCP, Azure, DigitalOcean e altri offrono tutti a terze parti la possibilità di affittare l'accesso alle risorse di calcolo in più posizioni geografiche.
```

Comment:
La prima parte è piuttosto ambigua anche in inglese, sembra accomunare le compute resources a network e disk... Io direi di tradurre così
```suggestion
Il _cloud computing_ è un modello che offre risorse di calcolo (come le CPU) e funzionalità di rete e di disco su richiesta, tramite Internet. Il cloud computing offre agli utenti la possibilità di accedere e utilizzare la potenza di calcolo in una posizione fisica remota. I provider cloud come AWS, GCP, Azure, DigitalOcean e altri offrono tutti a terze parti la possibilità di affittare l'accesso alle risorse di calcolo in più posizioni geografiche.
```